scripts/updateSeDrivers.sh

```diff
@@ -89,6 +89,9 @@ function setChromeDriverVersion() {
             78)
                 CHROME_DRIVER_VER=78.0.3904.70
             ;;
+            79)
+		        CHROME_DRIVER_VER=79.0.3945.36
```

Comment:
Is this relevant change? Is it required for testing?

Reply:
yes needed for testing because Chrome version is updated on travis